src/app.py

```diff
@@ -72,55 +69,55 @@ def get_response(user_input):
     
     return response['answer']
 
-# Web page setup
-# Basic page config
+# app config
 st.set_page_config(page_title="Chat with websites", page_icon="🤖")
+st.title("Chat with websites")
 
-st.title("RAG AI based chat with websites")
-
-# Left sidebar configuration to show logo & URL text box
+# sidebar
+website_url = 'https://toucharcade.com/'
```

Comment:
Should we change it to https://www.metacritic.com/game/elden-ring/

Reply:
It's already set to that